lib/xcprofiler/reporters/abstract_reporter.rb

```diff
@@ -46,5 +48,17 @@ def show_invalid_locations?
     def order
       options[:order] || :time
     end
+
+    def truncate_limit
+      options[:truncate_limit] ||= DEFAULT_TRUNCATE_LIMIT
+    end
+
+    def truncate(text)
```

Comment:
I prefer to declare this on `StandardOutputReporter` rather than here.

Because it is currently used just by that reporter.